x/examples/outline-connectivity-app/app_mobile/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "author": "Daniel LaCosse",
+  "dependencies": {
+    "@capacitor/core": "latest",
+    "@capacitor/ios": "^5.2.2",
+    "lit": "^2.7.6",
+    "shared_backend": "workspace:^",
+    "shared_frontend": "workspace:^"
+  },
+  "devDependencies": {
+    "@capacitor/cli": "latest",
+    "concurrently": "^8.2.0",
+    "typescript": "^5.1.6",
+    "vite": "^2.9.13"
+  },
+  "name": "app_mobile",
+  "scripts": {
+    "build": "concurrently 'yarn:build:*'",
```

Comment:
Is `concurrently` a preinstalled command line? Or is it included in `node_modules`?